frontend/common/services/useFeatureAnalytics.ts

```diff
@@ -0,0 +1,90 @@
+import { Res } from 'common/types/responses'
+import { Req } from 'common/types/requests'
+import { service } from 'common/service'
+import { sortBy } from 'lodash'
+import moment from 'moment'
+import range from 'lodash/range'
+
+export const featureAnalyticsService = service
+  .enhanceEndpoints({ addTagTypes: ['FeatureAnalytics'] })
+  .injectEndpoints({
+    endpoints: (builder) => ({
+      getFeatureAnalytics: builder.query<
+        Res['featureAnalytics'],
+        Req['getFeatureAnalytics']
+      >({
+        providesTags: [{ id: 'LIST', type: 'FeatureAnalytics' }],
```

Comment:
As discussed offline, for each combination of environments, the current state of the hook re-triggers a request for each of the environment while ideally we would cache each environment once and for all